services/api/src/routes/auth.js

```diff
@@ -19,11 +24,12 @@ router
       firstName: yd.string().required(),
       lastName: yd.string().required(),
       password: yd.string().password().required(),
+      phoneNumber: yd.string().phoneNumber().required(),
```

Comment:
how do i access the phoneNumber validation from the model here ? 

I was considering if could use the User.getCreateValidation ... but not sure that would fly, i am thinking its a bad idea because then you have to be real careful around what fields can be modified by the user.

Reply:
as discussed look for how the `validate: 'email'` field is handled and just add to that.

also I would prefer to call it "phone", the number part is implied